### PR TITLE
Reduce space used in glance

### PR DESCRIPTION
**Pull request by fgimenez**
Images uploaded to glance in qcow2 format; only keep 3 images for each triplet

### CODE REVIEW

File: cmd/snappy-cloud-image/main.go [hunk 1]
```diff
@@ -41,7 +41,7 @@ func main() {
 
 	imgDataOrigin := si.NewClient(httpClient)
 	imgDataTarget := cloud.NewClient(cliExecutor)
-	imgDriver := image.NewUDF(cliExecutor)
+	imgDriver := image.NewUDFQcow2(cliExecutor)
```
Comment (come-maiz) [inline cmd/snappy-cloud-image/main.go:44]: wow, this is a hard one. cow are also initials :)
Your option looks better than NewUDFQCOW2, though.

File: pkg/runner/runner.go [hunk 2]
```diff
@@ -145,26 +142,18 @@ func (r *Runner) cleanup(options *flags.Options) (err error) {
 	noDotsRelease := strings.Replace(options.Release, ".", "", 1)
 	imageList, err := r.imgDataTarget.GetVersions(noDotsRelease, options.Channel, options.Arch)
 	if err != nil {
+		log.Infof("Error getting image list")
```
Comment (come-maiz) [inline pkg/runner/runner.go]: Just Info here?
I'm not sure if it's needed, but I saw you changing it somewhere else.

**Comment by come-maiz** [comment]
lgtm.

File: pkg/runner/runner.go (hunk 2 above)
Comment (fgimenez) [inline pkg/runner/runner.go]: Good catch, fixed

**Comment by fgimenez** [comment]
Thanks

**Pull request merged**